### PR TITLE
Updating EntityBody[F] alias, to have the correct parameters

### PR DESCRIPTION
**Pull request by calvinbrown085**
Updated `EntityBody` to have the correct alias of ``Stream[F, Byte]` instead of `Stream[F, ByteVector]`

Updated links to the functional streams library and a youtube video on the topic.

closes https://github.com/http4s/http4s/issues/1505

### CODE REVIEW

File: docs/src/main/tut/entity.md [hunk 1]
```diff
@@ -12,8 +12,8 @@ streaming), but before the body has been fully received. The same applies to the
 http client usage, where you can start a connection before the body is fully
 materialized. You don't have to load the full body into memory to submit the
 request either. Taking a look at `Request[F]` and `Response[F]`, both have a body of
-type `EntityBody[F]`, which is simply an alias to `Stream[F, ByteVector]`. To
-understand `Stream`, take a look at the [streams-tutorial].
+type `EntityBody[F]`, which is simply an alias to `Stream[F, Byte]`. To
+understand `Stream`, take a look at the [streams-tutorial] or a youtube video [introduction-to-functional-streams].
```
Comment (ChristopherDavenport) [inline docs/src/main/tut/entity.md]: Don't think _a youtube video_ is necessary could probably remove so it reads _or introduction-to-functional-streams_

Comment (calvinbrown085) [reply docs/src/main/tut/entity.md]: @ChristopherDavenport 👍

**Comment by calvinbrown085** [comment]
@ChristopherDavenport Updated.

File: docs/src/main/tut/entity.md [hunk 2]
```diff
@@ -96,7 +96,7 @@ If you're working with [twirl] templates, there's a bridge for that too:
 
 - scala-twirl: `"org.http4s" %% "http4s-twirl" % Http4sVersion`
 
-[streams-tutorial]: https://gist.github.com/djspiewak/d93a9c4983f63721c41c
+[introduction-to-functional-streams]: https://github.com/functional-streams-for-scala/fs2
```
Comment (ChristopherDavenport) [inline docs/src/main/tut/entity.md]: This PR will be perfect with this line changed to 

```
[introduction-to-functional-streams]: https://youtu.be/cahvyadYfX8
```

Sorry for all the tiny changes!

Comment (calvinbrown085) [reply docs/src/main/tut/entity.md]: @ChristopherDavenport No problem!! Should be fixed now.

**Comment by ChristopherDavenport** [comment]
Thank you for your help! My apologies for the miscommunication, not my finest hour.

**Pull request merged**